todo/src/reducer/toDo.js

```diff
@@ -0,0 +1,15 @@
+import {ADD ITEM, TOOGLE ITEM} from 
+
+const toDo = (state =[], actions) => {
```

Comment:
Don't forget you need to be able to export these reducers. `export const toDo...`